src/libnvme.map

```diff
@@ -71,6 +71,7 @@ LIBNVME_1_0 {
 		nvme_flush;
 		nvme_format_nvm;
 		nvme_free_ctrl;
+		nvme_free_dirents;
```

Comment:
This should go into the LIBNVME_1_1 section

Reply:
nice catch! updated